truss/templates/shared/extra_config_resolver.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from typing import Optional
+
+
+class ExtraConfigResolver:
+    EXTRA_CONFIG_MOUNT_DIR = "/etc/extra_config"
+
+    @staticmethod
+    def get_config_value(config_key_name: str) -> Optional[str]:
```

Comment:
can this be accessed in a code path that needs to be fast? If so, reading from disk every time might be slow, so we should consider caching the config in memory for a short period

Reply:
We only access file contents from `/etc/dynamic_config` on chainlet initialization so we control this code path and will only read from disk once when a chainlet is deployed. We should be ok not caching the config for now.